tests/src/test/java/org/eclipse/hono/tests/registry/TenantAmqpIT.java

```diff
@@ -109,13 +138,24 @@ public void testGetTenant(final TestContext ctx) {
     @Test
     public void testGetNotConfiguredTenantReturnsUnauthorized(final TestContext ctx) {
 
-        tenantClient
-                .get("HTTP_ONLY")
+        final String tenantId = helper.getRandomTenantId();
+        final X500Principal subjectDn = new X500Principal("CN=ca, OU=Hono, O=Eclipse");
+        final TenantObject payload = new TenantObject()
+                .setTenantId(tenantId)
+                .setEnabled(true)
+                .setTrustAnchor(getRandomPublicKey(), subjectDn)
+                .addAdapterConfiguration( new JsonObject()
+                                .put(TenantConstants.FIELD_ADAPTERS_TYPE, Constants.PROTOCOL_ADAPTER_TYPE_HTTP)
+                                .put(TenantConstants.FIELD_ENABLED, true)
+                                .put(TenantConstants.FIELD_ADAPTERS_DEVICE_AUTHENTICATION_REQUIRED, true));
+
+        helper.registry.addTenant(JsonObject.mapFrom(payload))
+                .compose(r -> tenantClient.get(tenantId))
                 .setHandler(ctx.asyncAssertFailure(t -> {
-                    ctx.assertEquals(
-                            HttpURLConnection.HTTP_FORBIDDEN,
-                            ((ServiceInvocationException) t).getErrorCode());
-                }));
+                        ctx.assertEquals(
+                                HttpURLConnection.HTTP_FORBIDDEN,
```

Comment:
I am not sure if this test case should be included in the suite because it relies on the (static) configuration of the client's permissions in the AuthServer. So, unless you also can configure the permissions in AuthServer dynamically as part of the fixture, it probably doesn't make sense to include this test here.

Reply:
I think it make sense in the Integration Tests context as it reflects a sensible user story. Changing the Authentication HTTP client so we can dynamically create the complete fixture is definitely a good objective, and I wrote earlier, this could be dealt with in another PR. 
But in the meantime why decrease the test coverage ?

Reply:
I ageee with @jbtrystram. This is not a regression and we can tackle it in the future improvements.

Reply:
ok, then let's keep it for the time being